bin/cli.js

```diff
@@ -6,9 +6,19 @@ process.title = processTitle;
 
 var argv = require('yargs').argv;
 var init = require('./runner').init;
+var childProcess = require('child_process');
+var path = require('path');
 
 
 // stateful things
 var cliModel = init(argv);
 
-cliModel.run(cliModel);
+cliModel.run(cliModel, function() {
+  var exit = childProcess.spawnSync(path.join(__dirname, '../node_modules/.bin/elm-test'),
```

Comment:
Why not default to global elm-test? In CI, elm-test is often installed globally

Reply:
Or rather: fallback to global, prefer local

Reply:
sgtm!